poglink/config.py

```diff
@@ -30,7 +36,6 @@
 
 LIST_VALUES = [
     "allowed_roles",
-    "rates_urls",
```

Comment:
This is no longer a list type so I just removed it. Haven't looked too deply to make sure that's all I should do here though.